frontend/package.json

```diff
@@ -6,7 +6,7 @@
   "author": "Hexastack",
   "license": "AGPL-3.0-only",
   "scripts": {
-    "dev": "next dev -p 8081",
+    "dev": "next dev -p 8081 --turbopack",
     "build": "next build",
```

Comment:
I think the build should also be with turbopack : https://nextjs.org/docs/app/api-reference/turbopack

Reply:
Please ignore, still not stable for build.